### PR TITLE
Start on small multiple.

### PR DESCRIPTION
**Pull request by vlandham**
currently looks like:

screen shot 2016-09-02 at 3 01 30 pm

Probably needs baseline line and more styling.

### CODE REVIEW

File: src/containers/LocationPage/LocationPage.jsx [hunk 1]
```diff
@@ -376,11 +378,29 @@ class LocationPage extends PureComponent {
...
   renderCompareMetrics() {
+    const { timeSeriesStatus, locationTimeSeries, clientIspTimeSeries } = this.props;
+    let series = [];
+    if (locationTimeSeries) {
+      series = [locationTimeSeries].concat(clientIspTimeSeries);
+    }
+    const chartId = 'providers-small-mult';
+    const metrics = ['download_speed_mbps_median', 'upload_speed_mbps_median', 'rtt_avg', 'retransmit_avg'];
+    const metricNames = ['Download Speed', 'Upload Speed', 'RTT', 'Retransmission Rate'];
     return (
```
Comment (pbeshai) [inline src/containers/LocationPage/LocationPage.jsx]: Can you use the metrics object defined in constants.js?

Comment (vlandham) [inline src/containers/LocationPage/LocationPage.jsx]: sure!

**Comment by pbeshai** [comment]
+1 nice work Jim

**Comment by pbeshai** [comment]
Agreed it needs more styling and the base line, but we can polish it later

**Pull request merged**